docs/faq.md

```diff
@@ -75,13 +75,27 @@ to misunderstand configuration details that can have an impact on the results.
 If you can, please reach out to the authors to confirm your configuration looks
 good to them.
 
-## I'd like to get my fuzzer evaluated on the free FuzzBench service, but I don't want the results to be public yet.
-
-Please reach out to us at fuzzbench@google.com.
-
-## I'm working on a new tool, but I'm not ready to make my code public yet. Can I use the FuzzBench service?
-
-Yes, please reach out to us at fuzzbench@google.com.
+## I'd like to get my fuzzer evaluated, but I don't want the results and/or code to be public yet. Can I use the FuzzBench service?
+
+Probably yes. We run private experiments for this purpose.
+Please reach out to us at fuzzbench@google.com. If we agree to benchmark your
+fuzzer, please follow the guide on
+[adding a new fuzzer]({{ site.baseurl }}/getting-started/adding-a-new-fuzzer/)
+on how to integrate your fuzzer with FuzzBench.
+
+You can ignore the sections on [Requesting an experiment]({{ site.baseurl }}/getting-started/adding-a-new-fuzzer/#requesting-an-experiment) and
+[Submitting your integration]({{ site.baseurl }}/getting-started/adding-a-new-fuzzer/#submitting-your-integration).
+Please test your fuzzer builds and works with our benchmarks, we won't have
```

Comment:
nit, consider wording: "Please make sure that your fuzzer works with our benchmarks"